src/redux/dataSlice.tsx

```diff
@@ -1,12 +1,12 @@
-import { createSlice, createAsyncThunk } from '@reduxjs/toolkit'
+import { createSlice, createAsyncThunk, PayloadAction } from '@reduxjs/toolkit'
 import axios from 'axios'
+import { FormData } from '../types/types'
 
-const initialState = { loading: false, error: null }
+const initialState = { loading: false, error: null, active: 0 }
 
-export const sendData = createAsyncThunk('data/sendData', async (data: any) => {
+export const sendData = createAsyncThunk('data/sendData', async (data: FormData) => {
   try {
     await axios.post('https://api.sbercloud.ru/content/v1/bootcamp/frontend', data)
```

Comment:
put it in file api and create [axios instance](https://axios-http.com/docs/instance)